mypy/find_sources.py

```diff
@@ -102,11 +106,17 @@ def find_sources_in_dir(self, path: str) -> List[BuildSource]:
         seen = set()  # type: Set[str]
         names = sorted(self.fscache.listdir(path), key=keyfunc)
         for name in names:
-            # Skip certain names altogether
-            if name == '__pycache__' or name.startswith('.') or name.endswith('~'):
-                continue
```

Comment:
If default exclusions are always considered, we may need to preserve this code (also this doesn't consider `site-packages`, etc.).